source/frontend/src/features/documents/documentUpload/DocumentUploadForm.tsx

```diff
@@ -100,7 +101,7 @@ const DocumentUploadForm: React.FunctionComponent<IDocumentUploadFormProps> = pr
             <SectionField label="Document type" labelWidth="3" className="pb-2">
               <Select
                 data-testid="document-type"
-                placeholder="Select Document type"
+                placeholder={documentTypes.length > 1 ? 'Select Document type' : undefined}
```

Comment:
this doesn't end up printing 'undefined' does it?

Reply:
It does not. If placeholder is set to undefined, then it selects the first option